tests/cbor_decode/test1_suit_old_formats/prj.conf

```diff
@@ -5,4 +5,3 @@
 #
 
 CONFIG_ZTEST=y
-CONFIG_ZTEST_STACKSIZE=16384
```

Comment:
Why are these being removed?

Reply:
Because they were renamed recently in Zephyr (STACKSIZE -> STACK_SIZE) and I realized I don't actually need them.

Reply:
Added a sentence in the commit message.